### DISCUSSION TITLE
Access issue to HAL management console (redirect to port 9993 instead of 9990)

### DISCUSSION THREAD

**Discussion by TanabutT** [dcm4che/dcm4chee-arc-light#3849] [General]
**Describe the bug**
Access issue to HAL management console

1. after success run docker-compose up and can go to http://localhost:8080/dcm4chee-arc/ui2/study/study
2. Try to access Wildfly Administration Console: http://localhost:9990, 
3. it redirects to https://localhost:9993/console/index.html instead
4. try to log in with Username: root, Password: changeit. with no success
![image](https://user-images.githubusercontent.com/26840831/198078760-70073093-04a4-48ce-b1d3-5f6e8413e6cd.png)

 - OS: window 10
 - Browser chrome

What is the problem here: The user password or the authorization from a port in the local machine and the port outside the docker container?

**Comment by vrindanayak** [comment]
@TanabutT 

> it redirects to https://localhost:9993/console/index.html instead

This is expected. 

> try to log in with Username: root, Password: changeit. with no success

By default, Wildfly admin user is not created, explicitly set it in env variables. See
- https://github.com/dcm4che-dockerfiles/dcm4chee-arc-psql#wildfly_admin_user-ignored-by-archive-versions-secured-by-keycloak-and-wildfly_admin_oidctrue
- https://github.com/dcm4che-dockerfiles/dcm4chee-arc-psql#wildfly_admin_password-ignored-by-archive-versions-secured-by-keycloak-and-wildfly_admin_oidctrue
